src/vellum/workflows/events/types.py

```diff
@@ -49,7 +44,7 @@ def default_serializer(obj: Any) -> Any:
 
 class BaseParentContext(UniversalBaseModel):
     span_id: UUID
-    parent: Optional['ParentContext'] = None
+    parent: Optional["ParentContext"] = None
```

Comment:
[oos] looks like we need some linting in CI 😄

Reply:
that's the black/flake task I had on my list from yesterday haha